{
  "advisories": [],
  "affected": "1.9.0-3",
  "fixed": "1.10.0-1",
  "issues": [
    "CVE-2020-29651"
  ],
  "name": "AVG-1338",
  "packages": [
    "python-py"
  ],
  "severity": "Low",
  "status": "Fixed",
  "ticket": null,
  "type": "denial of service"
}
